Issue #129 - fix TimeExtensions feature bug

chrislwade opened the pull request:
- Switch 'Scenario: DateTime Accuracy to One Second' into an outline.
- Use explicit `DateTime` values to control random failures.

jbatte47 commented on src/_specs/Steps/Runtime/TimeExtensionsSteps.cs:
```diff
@@ -44,16 +44,13 @@ public TimeExtensionsSteps(TimeExtensionsContext context)
+				_context.SecondValue = dateTime;
+			else
+				_context.FirstValue = dateTime;
 		}
```
The above assignments should swap; currently the following step:

``` gherkin
Given I have a second DateTime representation of 1/1/2000 00:00:01.050
```

Assigns Y2K to `FirstValue`, not `SecondValue`. A small point since it's a value pair that only gets used for comparison, hence the tests all pass. Just my 2 neurotic cents :)

chrislwade committed:
Issue #129 - fix TimeExtensions feature bug

- Switch 'Scenario: DateTime Accuracy to One Second' into an outline.
- Use explicit `DateTime` values to control random failures.

chrislwade commented:
Fixed.
